lib/board.rb

```diff
@@ -0,0 +1,61 @@
+class Board
+  attr_accessor :cells
+
+  def initialize
+    reset!
+  end
+
+  def cells
+    @cells
+  end
+
+  def reset!
+    self.cells=(Array.new(9, " "))
+  end
+
+  def display #Prints the Board
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(position) #Takes in user input & returns the value of the board cell
+    index = input_to_index(position)
+    cells[index]
+  end
+
+  def full?
+    cells.all? {|index| index == "X" || index == "O"}
+  end
+
+  def turn_count #Returns the amount of turns
```

Comment:
Use the highest level and most expressive iterator possible to solve a problem. [`count`](http://ruby-doc.org/core-2.2.0/Array.html#method-i-count) on the cells is perfect and the rest of this method is not needed. Entire body could read:

```ruby
def turn_count
  cells.count { |cell| cell == "X" || cell == "O" }
end
```